src/baldr/graphreader.cc

```diff
@@ -22,32 +22,43 @@ constexpr size_t AVERAGE_MM_TILE_SIZE = 1024;         // 1k
 namespace valhalla {
 namespace baldr {
 
-struct GraphReader::tile_extract_t : public midgard::tar {
-  tile_extract_t(const boost::property_tree::ptree& pt)
-      : tar(pt.get<std::string>("tile_extract", "")) {
+struct GraphReader::tile_extract_t {
+  tile_extract_t(const boost::property_tree::ptree& pt) {
     // if you really meant to load it
     if (pt.get_optional<std::string>("tile_extract")) {
-      // map files to graph ids
-      for (auto& c : contents) {
-        try {
-          auto id = GraphTile::GetTileId(c.first);
-          tiles[id] = std::make_pair(const_cast<char*>(c.second.first), c.second.second);
-        } catch (...) {}
-      }
-      // couldn't load it
-      if (tiles.empty()) {
-        LOG_WARN("Tile extract could not be loaded");
-      } // loaded ok but with possibly bad blocks
-      else {
-        LOG_INFO("Tile extract successfully loaded");
-        if (corrupt_blocks) {
-          LOG_WARN("Tile extract had " + std::to_string(corrupt_blocks) + " corrupt blocks");
+      try {
```

Comment:
mostly an indent change here, but instead of deriving from tar we encapsulate it and then if it fails to load we log